modules/installation-aws-limits.adoc

```diff
@@ -16,30 +16,29 @@ to request additional resources for you AWS account.
 The following table summarizes the AWS components whose limits can impact your
 ability to install and run {product-title} clusters.
 
-
 [cols="2a,3a,3a,8a",options="header"]
 |===
 |Component |Number of clusters available by default| Default AWS limit |Description
 
-|Instance Limits
-|Varies
-|Varies
+|vCPU limits
+|0
+|1
```

Comment:
Here "Number of clusters available by default" - 0, "Default AWS limit" is right? Or just some place holder? I think we should input some meaningful value here.